contracts/solve/src/ERC7683/SolverNetInbox.sol

```diff
@@ -80,24 +80,12 @@ contract SolverNetInbox is OwnableRoles, ReentrancyGuard, Initializable, Deploye
      * @notice Returns the order with the given ID.
      * @param id  ID of the order.
      */
-    function getOrder(bytes32 id) external view returns (SolverNetOrder memory) {
-        return SolverNetOrder({ order: _orders[id], params: _orderParams[id], history: _orderHistory[id] });
-    }
-
-    /**
-     * @notice Returns the order parameters for the given order ID.
-     * @param id  ID of the order.
-     */
-    function getOrderParams(bytes32 id) external view returns (SolverNetOrderParams memory) {
-        return _orderParams[id];
-    }
-
-    /**
-     * @notice Returns the order history for the given order ID.
-     * @param id  ID of the order.
-     */
-    function getOrderHistory(bytes32 id) external view returns (StatusUpdate[] memory) {
-        return _orderHistory[id];
+    function getOrder(bytes32 id)
+        external
+        view
+        returns (ResolvedCrossChainOrder memory order, OrderState memory state, StatusUpdate[] memory history)
+    {
+        return (_orders[id], _orderState[id], _orderHistory[id]);
     }
```

Comment:
Struggled thinking of a more appropraite "wrapping" struct name. Everything felt forced. I actually think `SolvertNetOrder` is decent, in that a `SolverNetOrder` wraps a `ResolvedCrossChainOrder`

But didn't love that we us overloading the use of "order". And we just needed this to name a return type. Tuple should suffice, as this is view is mostly for off chain and easy to unpack.